sdk/ai/azopenai/autorest.md

```diff
@@ -294,4 +303,32 @@ directive:
       return $
         .replace(/populate\(objectMap, "model", (c|e).Model\)/g, 'populate(objectMap, "model", &$1.Deployment)')
         .replace(/err = unpopulate\(val, "Model", &(c|e).Model\)/g, 'err = unpopulate(val, "Model", &$1.Deployment)');
+
+  # - from: client.go
```

Comment:
Can this be deleted or is it temporary?

Reply:
Yeah, you're right. The directives below handle the real renaming now (there was an awkward intermediate state here).